.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/google/yapf
+    rev: 'v0.31.0'
+    hooks:
+      - id: yapf
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: '3.9.1'
+    hooks:
+      - id: flake8
+        args:
+        - --show-source
+        additional_dependencies:
+          - pep8-naming
```

Comment:
Are there other flake8 plugins that we should be using?

Reply:
Addressed in top-level PR comment.